Protecs/src/main/java/uk/ac/ucl/protecs/sim/LoadPopulation.java

```diff
@@ -91,13 +94,16 @@ public static void load_population(String agentsFilename, WorldBankCovid19Sim si
 					rawHouseholds.put(hhName, h);
 					sim.households.add(h);
 				}
-				// if the workplace doesn't already exist, create it and save it
-				if(w == null){
+				if (usingWorkplaces) {
```

Comment:
we could join this into if(usingWorkplaces && w == null) to skip an extra statement!